protostar/commands/migrate_command.py

```diff
@@ -100,6 +108,7 @@ async def migrate(
         output_dir_path: Optional[Path],
         migrator_config: MigratorExecutionEnvironment.Config,
         no_confirm: bool,
+        compiled_contracts_dir: Path,
```

Comment:
[Consistency] `output_dir_path` above uses different naming convention for the same thing

```suggestion
        output_dir_path: Optional[Path],
        migrator_config: MigratorExecutionEnvironment.Config,
        no_confirm: bool,
        compiled_contracts_dir_path: Path,
```